driver/kubernetes/driver.go

```diff
@@ -41,15 +41,16 @@ type Driver struct {
 
 	// if you add fields, remember to update docs:
 	// https://github.com/docker/docs/blob/main/content/build/drivers/kubernetes.md
-	minReplicas      int
-	deployment       *appsv1.Deployment
-	configMaps       []*corev1.ConfigMap
-	clientset        *kubernetes.Clientset
-	deploymentClient clientappsv1.DeploymentInterface
-	podClient        clientcorev1.PodInterface
-	configMapClient  clientcorev1.ConfigMapInterface
-	podChooser       podchooser.PodChooser
-	defaultLoad      bool
+	minReplicas         int
+	deployment          *appsv1.Deployment
+	configMaps          []*corev1.ConfigMap
+	clientset           *kubernetes.Clientset
+	deploymentClient    clientappsv1.DeploymentInterface
+	podClient           clientcorev1.PodInterface
+	configMapClient     clientcorev1.ConfigMapInterface
+	podChooser          podchooser.PodChooser
+	defaultLoad         bool
+	provisioningTimeout int
```

Comment:
I'm not sure on naming it `provisioningTimeout` as this is applied in a retry logic. Maybe `retryTimeout`?